networkx/algorithms/flow/tests/test_mincost.py

```diff
@@ -29,6 +29,24 @@ def test_simple_digraph(self):
         assert_equal(nx.cost_of_flow(G, H), 24)
         assert_equal(H, soln)
 
+    def test_floats_digraph(self):
+        G = nx.Graph()
+        G.add_nodes_from(['i', 'j', 'k', 'l'])
+        G.node['i']['demand'] = -1
+        G.node['j']['demand'] = 1. / 3
```

Comment:
Instead of using `1. / 3`, use `1 / 3` and put `from __future__ import division` before all other imports.